nixnet/system/databases.py

```diff
@@ -0,0 +1,147 @@
+            yield alias
+
+    def values(self):
+        # type: () -> typing.Iterable[Database]
+        """Return all Database objects in the system.
+
+            Yields:
+                An iterator to all the values in the Databases object.
+        """
+        for alias, filepath in _funcs.nxdb_get_database_list(''):
+            yield self._create_item(self._handle, alias, alias, filepath)
+
+    def items(self):
+        # type: () -> typing.Iterable[(str, Database)]
+        """Return all aliases and database objects associated with those aliases in the system.
+
+            Yields:
+                An iterator to tuple pairs of alias and database objects in the system.
+        """
+        for alias, filepath in _funcs.nxdb_get_database_list(''):
+            yield alias, self._create_item(self._handle, alias, alias, filepath)
+
+    def add_alias(self, database_alias, database_filepath, default_baud_rate):
+        # type: (str, str, int) -> None
+        """Add a new alias with baud rate size of up to 64 bits to a database file.
+
+        NI-XNET uses alias names for database files. The alias names provide a
+        shorter name for display, allow for changes to the file system without
+        changing the application, and enable efficient deployment to LabVIEW
+        Real-Time (RT) targets.
+
+        This function is supported on Windows only. For RT targets, you can pass
+        the new alias to 'deploy' to transfer an optimized binary image of the
+        database to the RT target. After deploying the database, you can use the
+        alias name in any application for the Windows host and RT target.
+
+        Args:
+            database_alias(str): Provides the desired alias name. Unlike the names of
+                other XNET database objects, the alias name can use special
+                characters such as space and dash. Commas are not allowed in the
+                alias name. If the alias name already exists, this function
+                changes the previous filepath to the specified filepath.
+            database_filepath(str): Provides the path to the CANdb, FIBEX, or LDF
+                file. Commas are not allowed in the alias name.
+            default_baud_rate(int):
+                Provides the default baud rate, used when filepath refers to a
+                CANdb database (.dbc) or an NI-CAN database (.ncd). These
+                database formats are specific to CAN and do not specify a cluster
+                baud rate. Use this default baud rate to specify a default CAN
+                baud rate to use with this alias. If database_filepath refers to
+                a FIBEX database (.xml) or LIN LDF file, the default_baud_rate
+                parameter is ignored. The FIBEX and LDF database formats require
+                a valid baud rate for every cluster, and NI-XNET uses that baud
+                rate as the default.
```

Comment:
I think we talked about defaulting `default_baud_rate` to `None`.

- If its `None` and a file type that doesn't need it, we just pass 0.
- If its `None` and is file type that needs it, we throw.
- If its not `None` and a file type that doesn't need it, we error.

Granted, there are pros/cons to this.  I'm still torn on it myself.